lib/configuration.js

```diff
@@ -58,6 +58,11 @@ export default class Configuration {
     return Sequence.readMigrations(this.fs, absolute, first, last);
   }
 
+  latestMigration(relative) {
+    const absolute = path.resolve(this.directory, relative);
```

Comment:
This is a giant chain of functions calling functions calling functions to finally get to the right level that actually does the thing and gets me the version.